tracing-journald/src/memfd.rs

```diff
@@ -8,7 +8,13 @@ use std::os::raw::c_uint;
 use std::os::unix::prelude::{FromRawFd, RawFd};
 
 fn create(flags: c_uint) -> Result<File> {
-    let fd = unsafe { memfd_create("tracing-journald\0".as_ptr() as *const c_char, flags) };
+    let fd = unsafe {
```

Comment:
nit, take it or leave it: it may be worth a comment explaining why we make the syscall ourselves instead of using libc? not a big deal, but it could be worth writing down for the future.